apps/website/src/pages/docs/tabs.mdx

```diff
@@ -9,27 +9,92 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tabs' />
-
 <LiveExample src='Tabs.main.tsx'>
   <AllExamples.TabsMainExample client:load />
 </LiveExample>
 
-## Usage
+### Composition API
+
+You can build Tabs with subcomponents that are fully customizable.
+
+We provide:
+
+- `Tabs.Wrapper`
+- `Tabs.List`
+- `Tabs.Tab`
+- `Tabs.TabIcon`
+- `Tabs.TabLabel`
+- `Tabs.TabDescription`
```

Comment:
we should explain the convenience props in `Tabs.Tab` (e.g. `label`) and the subcomponents api

Reply:
Added a small section